roles/add-vhost/tasks/main.yml

```diff
@@ -50,7 +52,7 @@
     force_basic_auth: yes
     body_format: json
     body:
-      domain_name: "{{ inventory_hostname }}{{ stanford_environment }}.cardinalsites.stanford.edu"
+      domain_name: "{{ inventory_hostname }}{{ stanford_environment }}.{{ sites_service }}.stanford.edu"
```

Comment:
We may as well define the full url path in our configuration instead of assuming `.stanford.edu` as we have already gone as far as to create a variable for it.

Reply:
I think we're using `sites_service` elsewhere. Although I suppose we could append `.stanford.edu` in the definition of the variable.